views/index.jade

```diff
@@ -23,19 +23,21 @@ block content
     div.container
       div.row
         div.col-md-6.whatis
-          h2 Build unstoppable applications
+          h2 Welcome to the Ethereum open source community
 
-          p Ethereum is a &nbsp;
-            strong decentralized platform that runs smart contracts
-            | : applications that run exactly as programmed without any possibility of downtime, censorship, fraud or third party interference.
+          p Ethereum is an open-source, public, blockchain-based distributed computing platform featuring smart contracts that
+            | facilitate online agreements. The Ethereum Virtual Machine (EVM)
+            | executes scripts on a decentralized network of servers (nodes) and is fueled by its virtual currency (ether).
 
-          p These apps run on a custom built &nbsp;
-            strong blockchain, an enormously powerful shared global infrastructure that can move value around and represent the ownership of property.&nbsp;
-            | This enables developers to create markets, store registries of debts or promises, move funds in accordance with instructions given long in the past (like a will or a futures contract) and many other things that have not been invented yet, all without a middle man or counterparty risk.
+          p With Ethereum you can build decentralized applications called dapps to
+            | create markets, store registries, raise funds, lend money, prove existence, create cryptocurrencies,
+            | run social media sites, bank the unbanked, unbank the banked, run prediction services, secure digital rights, and more.
```

Comment:
How close are we here?   If we make the decentralization image and text into a new section, are we ready to move forward?  If so, I'll add that to the PR.